docs/source/serenata_toolbox.rst

```diff
@@ -1,41 +1,14 @@
 serenata_toolbox package
 ========================
 
-Submodules
-----------
+Subpackages
```

Comment:
AFAIK they are submodules, not subpackages… but I assume this is merely 💅 So the real question is: have you changed it yourself or was it changed automagically? If it was automagically just let it be ; )